src/main/java/software/amazon/lambda/snapstart/CacheLambdaHandlerFields.java

```diff
@@ -0,0 +1,47 @@
+package software.amazon.lambda.snapstart;
+
+import edu.umd.cs.findbugs.BugReporter;
+import edu.umd.cs.findbugs.Detector;
+import edu.umd.cs.findbugs.ba.ClassContext;
+import edu.umd.cs.findbugs.ba.XClass;
+import org.apache.bcel.classfile.Field;
```

Comment:
This import should be sorted (after all the `edu.umd..`) (We probably need some sort of checkstyle to check for this in the future)

Reply:
Updated. Thanks